recipe/meta.yaml

```
@@ -297,12 +297,20 @@ outputs:
         Activation for Intel® oneAPI DPC++/C++ Compiler.
         This package is a repackaged set of binaries obtained directly from Intel's Anaconda.org channel.
     test:
+      requires:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      files:
+        - simple.cpp
       commands:
         - ls -A1 ${PREFIX}/bin/*         # [unix]
         - ls -A1 ${PREFIX}/include/*     # [unix]
         - ls -A1 ${PREFIX}/lib/*         # [unix]
         - dir %PREFIX%\Library\bin\*  # [win]
         - dir %PREFIX%\Library\lib\*  # [win]
+        - dpcpp --gcc-toolchain=$PREFIX --sysroot=$PREFIX/$BUILD/sysroot -target $BUILD simple.cpp -lpthread -o simple.exe  # [unix]
```

@isuruf Can you look at this? I think this line needs CXXFLAGS and LDFLAGS maybe?

```suggestion
        - dpcpp --gcc-toolchain=$PREFIX --sysroot=$PREFIX/$BUILD/sysroot -target $BUILD ${LDFLAGS} ${CXXFLAGS} simple.cpp -lpthread -o simple.exe  # [unix]
```